### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
This is landing page of my Component Library
please give it a review

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]

File: Css/main.css [hunk 1]
```diff
@@ -148,4 +152,49 @@ footer{
...
+    font-family: var(--designed-font);
+    font-size:2.5rem;
+    padding:1rem .5rem;
+}
+
+.index-div{
+    display: flex;
+    padding: 2rem 2rem;
+    flex-flow: row wrap;
+    background: #f4f2f8;
```
Comment (msdeshmukh009) [inline Css/main.css:169]: A color variable here may be useful in the future.

File: Css/main.css [hunk 2]
```diff
@@ -148,4 +152,49 @@ footer{
...
+    margin:1rem;
+    text-decoration: none;
+}
+
+
+.setup{
+    display: flex;
+    padding: 2rem 2rem;
+    flex-flow: column wrap;
+    background: #f4f2f8;
```
Comment (msdeshmukh009) [inline Css/main.css:195]: color variable maybe?

**Review by msdeshmukh009 — COMMENTED** [review]
I think you have some issue going on with git as I see many similar changes in both PR so maybe resolve them first before merging.

**Comment by Varu98** [comment]
Good Work on the Barrel Imports for the CSS files. 👍🏻

**Pull request merged**